packages/data-point/lib/entity-types/base-entity/factory.js

```diff
@@ -40,9 +87,67 @@ function create (Factory, spec, id) {
     entity.outputType = createReducer(outputType)
   }
 
-  entity.params = deepFreeze(defaultTo(spec.params, {}))
+  entity.params = defaultTo(spec.params, {})
```

Comment:
why not just `spec.params || {}`? lodash doesn't seem necessary here

Reply:
honestly, .... so I dont get one more branch to tests.... is this cheating? :)

Reply:
ha! if the tests are something we need to circumvent, then 100% branch coverage is probably not something we need to care about...

Reply:
I know I know :)